pyproject.toml

```diff
@@ -61,7 +61,3 @@ implicit_reexport = true
 strict_equality = true
 
 warn_unused_configs = true
-
-[[tool.mypy.overrides]]
-module = ["cunumeric.array"]
-ignore_errors = true
```

Comment:
We could leave an empty list here as a convenience in case new modules need to be temp ignored in the future